stglib/iq.py

```diff
@@ -20,8 +20,13 @@ def mat_to_cdf(metadata):
 
     del metadata
 
+    ds = create_iqbindist(ds)
+
     ds = utils.ensure_cf(ds)
 
+    # Compute time stamps
+    # ds = utils.shift_time(ds, ds.attrs["flowSampleDuration"] / 2)
```

Comment:
Looks like this was added but as a comment only. Is this correct?

Reply:
Yes, I saw this was in the aqd hdr2cdf.py. I was going to use it but then decided otherwise. Do you think I should include this or delete?